api/src/project_view_details.ts

```diff
@@ -77,11 +77,13 @@ function mkSwaggerSchema(server: FastifyInstance) {
                       type: "array",
                       items: {
                         type: "object",
+                        additionalProperties: true,
```

Comment:
why?

Reply:
you are right, TraceEvents can only look like this and do not allow any other properties